test/goldens/end2end/webdriver-3.0.0.json_report.md

```diff
@@ -0,0 +1,82 @@
+## 20/20 Follow Dart file conventions
+
+### [*] 10/10 points: Provide a valid `pubspec.yaml`
+
+
+### [*] 5/5 points: Provide a valid `README.md`
+
+
+### [*] 5/5 points: Provide a valid `CHANGELOG.md`
+
+
+## 0/10 Provide documentation
+
+### [x] 0/10 points: Package has an example
+
+<details>
+<summary>
+No example found.
+</summary>
+
+See [package layout](https://dart.dev/tools/pub/package-layout#examples) guidelines on how to add an example.
+</details>
+
+## 0/20 Support multiple platforms
+
+### [x] 0/20 points: Supports 0 of 2 possible platforms (native, js)
+
+
+Consider supporting these prerelease platforms:
+
+<details>
+<summary>
+Package not compatible with SDK dart
+</summary>
+
+Because it is not compatible with any of the supported runtimes: native-aot, native, js
```

Comment:
We should end with a full stop `.`.

Reply:
We really should document somewhere what this means, and link.
As the package author there is not much to go on.